base/stats.go

```diff
@@ -232,31 +232,32 @@ type CBLReplicationPushStats struct {
-	DocWritesBytesBlip      *SgwIntStat `json:"doc_writes_bytes_blip"`
-	DocWritesXattrBytes     *SgwIntStat `json:"doc_writes_xattr_bytes"`
-	HighSeqFeed             *SgwIntStat `json:"high_seq_feed"`
-	NumDocReadsBlip         *SgwIntStat `json:"num_doc_reads_blip"`
-	NumDocReadsRest         *SgwIntStat `json:"num_doc_reads_rest"`
-	NumDocWrites            *SgwIntStat `json:"num_doc_writes"`
-	NumReplicationsActive   *SgwIntStat `json:"num_replications_active"`
-	NumReplicationsTotal    *SgwIntStat `json:"num_replications_total"`
-	NumTombstonesCompacted  *SgwIntStat `json:"num_tombstones_compacted"`
-	SequenceAssignedCount   *SgwIntStat `json:"sequence_assigned_count"`
-	SequenceGetCount        *SgwIntStat `json:"sequence_get_count"`
-	SequenceIncrCount       *SgwIntStat `json:"sequence_incr_count"`
-	SequenceReleasedCount   *SgwIntStat `json:"sequence_released_count"`
-	SequenceReservedCount   *SgwIntStat `json:"sequence_reserved_count"`
-	WarnChannelsPerDocCount *SgwIntStat `json:"warn_channels_per_doc_count"`
-	WarnGrantsPerDocCount   *SgwIntStat `json:"warn_grants_per_doc_count"`
-	WarnXattrSizeCount      *SgwIntStat `json:"warn_xattr_size_count"`
+	ConflictWriteCount       *SgwIntStat `json:"conflict_write_count"`
+	Crc32MatchCount          *SgwIntStat `json:"crc32c_match_count"`
+	DCPCachingCount          *SgwIntStat `json:"dcp_caching_count"`
+	DCPCachingTime           *SgwIntStat `json:"dcp_caching_time"`
+	DCPReceivedCount         *SgwIntStat `json:"dcp_received_count"`
+	DCPReceivedTime          *SgwIntStat `json:"dcp_received_time"`
+	DocReadsBytesBlip        *SgwIntStat `json:"doc_reads_bytes_blip"`
+	DocWritesBytes           *SgwIntStat `json:"doc_writes_bytes"`
+	DocWritesBytesBlip       *SgwIntStat `json:"doc_writes_bytes_blip"`
+	DocWritesXattrBytes      *SgwIntStat `json:"doc_writes_xattr_bytes"`
+	HighSeqFeed              *SgwIntStat `json:"high_seq_feed"`
+	NumDocReadsBlip          *SgwIntStat `json:"num_doc_reads_blip"`
+	NumDocReadsRest          *SgwIntStat `json:"num_doc_reads_rest"`
+	NumDocWrites             *SgwIntStat `json:"num_doc_writes"`
+	NumReplicationsActive    *SgwIntStat `json:"num_replications_active"`
+	NumReplicationsTotal     *SgwIntStat `json:"num_replications_total"`
+	NumTombstonesCompacted   *SgwIntStat `json:"num_tombstones_compacted"`
+	SequenceAssignedCount    *SgwIntStat `json:"sequence_assigned_count"`
+	SequenceGetCount         *SgwIntStat `json:"sequence_get_count"`
+	SequenceIncrCount        *SgwIntStat `json:"sequence_incr_count"`
+	SequenceReleasedCount    *SgwIntStat `json:"sequence_released_count"`
+	SequenceReservedCount    *SgwIntStat `json:"sequence_reserved_count"`
+	WarnChannelsPerDocCount  *SgwIntStat `json:"warn_channels_per_doc_count"`
+	WarnGrantsPerDocCount    *SgwIntStat `json:"warn_grants_per_doc_count"`
+	WarnXattrSizeCount       *SgwIntStat `json:"warn_xattr_size_count"`
+	WarnChannelNameSizeCount *SgwIntStat `json:"warn_channel_name_size_count"`
```

Comment:
Minor comment but might be nice if we can keep these alphabetically ordered, given the number of stats it makes it easier to scan over